pkgs/applications/graphics/mcomix3/default.nix

```diff
@@ -0,0 +1,90 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, python3
+, python3Packages
+, wrapGAppsHook
+, gobject-introspection
+, gtk3
+, gdk-pixbuf
+# Recommended Dependencies:
+, unrarSupport ? false
+, unrar
+, p7zip
+, lhasa
+, mupdf
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "mcomix3";
+  version = "20201123";
```

Comment:
According to https://nixos.org/nixpkgs/manual/#sec-package-naming, the `version` should be `unstable-yyyy-mm-dd`, when using git snapshots.

Reply:
Okay, everybody is telling me something different about what the version should be. So far, I had `1.2.1+git20201123`, `v1.2.1+git<commit>`, `20201123` your `unstable-2020-11-23`. I'll take yours, because you provided a manual link.